src/components/widgets/LabelTextarea.vue

```diff
@@ -40,6 +40,7 @@ const elTextArea = ref();
         :ref="elTextArea"
         class="form-control col-9 d-inline-block"
         :class="{ 'is-invalid': error != null }"
+        :placeholder="placeholder"
```

Comment:
I think this is handled by line 39 `v-bind="$attrs"` but no harm in adding this explicitly

Reply:
Amended on [🗑️ Remove redundant placeholder bind](https://github.com/RedHatProductSecurity/osim/pull/116/commits/f2da86ba15b7ca3b20273f419e8ab0dd9ad08ae5)